package.json

```diff
@@ -1,24 +1,107 @@
 {
   "name": "videojs-vimeo",
-  "version": "2.0.2",
-  "description": "Allows you to use Vimeo URL as source with Video.js.",
-  "main": "src/Vimeo.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/eXon/videojs-vimeo.git"
+    "url": "https://github.com/didacte/videojs-vimeo.git"
+  },
+  "version": "3.0.0",
+  "description": "Videojs Vimeo plugin using the official Vimeo Player API.",
+  "main": "es5/Vimeo.js",
+  "scripts": {
+    "prebuild": "npm run clean",
+    "build": "npm-run-all -p build:*",
+    "build:js": "npm-run-all build:js:babel build:js:browserify build:js:bannerize build:js:collapse build:js:uglify",
+    "build:js:babel": "babel src -d es5",
+    "build:js:bannerize": "bannerize dist/videojs-vimeo.js --banner=scripts/banner.ejs",
+    "build:js:browserify": "browserify . -g browserify-shim -s videojs-vimeo -o dist/videojs-vimeo.js",
+    "build:js:collapse": "bundle-collapser dist/videojs-vimeo.js -o dist/videojs-vimeo.min.js",
+    "build:js:uglify": "uglifyjs dist/videojs-vimeo.min.js --comments --mangle --compress -o dist/videojs-vimeo.min.js",
+    "clean": "rimraf dist test/dist es5 && mkdirp dist test/dist es5",
+    "lint": "vjsstandard",
+    "start": "babel-node scripts/server.js",
+    "version": "babel-node scripts/version.js",
+    "postversion": "babel-node scripts/postversion.js",
+    "prepublish": "npm run build"
   },
   "keywords": [
-    "video.js",
-    "vimeo",
-    "plugin"
+    "videojs",
+    "videojs-plugin",
+    "vimeo"
   ],
   "author": "Benoit Tremblay <trembl.ben@gmail.com>",
   "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/eXon/videojs-vimeo/issues"
+  "browserify": {
+    "transform": [
+      "browserify-versionify"
+    ]
+  },
+  "browserify-shim": {
+    "qunit": "global:QUnit",
+    "sinon": "global:sinon",
+    "video.js": "global:videojs"
```

Comment:
Shimming video.js via browserify-shim is what's causing the `Cannot read property 'getComponent' of undefined` mentioned above.

> Makes define and also module be undefined, in order to fix improperly-authored libraries that need shimming but try anyway to use AMD or CommonJS. For more info read the comment inside this fixture
https://github.com/thlorenz/browserify-shim